src/time_evolution/mcsolve.jl

```diff
@@ -23,6 +23,7 @@ end
 function LindbladJumpAffect!(integrator)
     internal_params = integrator.p
     c_ops = internal_params.c_ops
+    c_ops_herm = internal_params.c_ops_herm
```

Comment:
I thought this is done by another PR, maybe you need to merge from main branch locally first.